src/Files.App/Views/PaneHolderPage.xaml.cs

```diff
@@ -49,32 +49,18 @@ public TabItemArguments TabItemArguments
 
 		private bool windowIsCompact
 		{
-			get
-			{
-				return _windowIsCompact;
-			}
+			get => _windowIsCompact;
 			set
 			{
 				if (value != _windowIsCompact)
 				{
 					_windowIsCompact = value;
-					if (value)
```

Comment:
This does not look equivalent, could you check?

Reply:
Thank you for letting me notice.
I'll revert these changes